src/ParseServer.js

```diff
@@ -98,10 +98,37 @@ class ParseServer {
       addParseCloud();
       if (typeof cloud === 'function') {
         cloud(Parse);
-      } else if (typeof cloud === 'string') {
-        require(path.resolve(process.cwd(), cloud));
+        return;
+      }
+      let cloudFiles = [];
+      if (typeof cloud === 'string') {
+        cloudFiles.push(cloud);
+      } else if (cloud instanceof Array) {
+        cloudFiles = cloud;
       } else {
-        throw "argument 'cloud' must either be a string or a function";
+        throw "argument 'cloud' must either be a string, array, or function";
+      }
+      for (const cloudFileName of cloudFiles) {
+        const cloudDir = fs.lstatSync(cloudFileName);
+        const filePath = path.resolve(process.cwd(), cloudFileName);
+        const requireFiles = files => {
+          const ext = path.extname(files) || '';
+          if (ext != '.js') {
```

Comment:
!==

Reply:
I'm actually not sure about this verification. It may make sense when you set a folder, but if, for any reason, someone specifies a single file with other extension, I believe we should try to require.